### ISSUE TITLE
Strange issue when importing exported segments to Mac iMovie

### ISSUE THREAD

**Issue by bouscram** [mifi/lossless-cut#519]
- [x] I have tested the newest version
- [x] I have checked [readme](https://github.com/mifi/lossless-cut) and **Known issues** / **Troubleshooting**
- [x] I have searched for similar issues

When I import an extract from a mp4 file in Mac iMovie (version 10.2.1), it always appears as there is no sound except if the beginning of the extract is the beginning of the original file. I have tried several files from different origins, and the behaviour has always been the same.
I have also tried ffmpeg directly with the additional  `-ss '00.00'` command (as it is not in the command used by LosslessCut), but the result is the same.

I suspect an iMovie bug (because the files indeed have sound), but maybe there is something in the files themselves which could be corrected by additional ffmpeg options.  

Naturally I can submit files examples if wanted, but I think any .mp4 file will do to reproduce the problem.

**Comment by bouscram** [comment]
Update: I have remuxed my files (with _remux_), and the sound shows in iMovie... So there is definitely something in the files formatting. _remux_ uses also ffmpeg, so it must be an option missing or something similar.

**Comment by bouscram** [comment]
Update 2: I had carefully set the cut points at keyframes, so I had not taken the "cut" setting value, which was at the default "normal cut".  If I use the "cut at keyframe" option, I have no issue with the sound in iMovie. Also, I have remarked that the files I have remuxed have a black part at the beginning (with sound in it), which seems to show that the cut was not at a keyframe... Could it be an error in the calculation of the time passed to ffmpeg?
